### PR TITLE
fix: Fix typo for WebpackCLI class

### PR DESCRIPTION
**Pull request by godcrampy**
**What kind of change does this PR introduce?**

- bug-fix

**Did you add tests for your changes?**
- Not relevant

**If relevant, did you update the documentation?**
- Not relevant

**Summary**
Fixed the following two typos:
1. `lib/webpack-cli.js`, on line 14 the CLI class is defined as:
```class WebpackCLI extends GroupHelper{```
but on line 180, it was exported as
```module.exports = webpackCLI;```

2. `lib/bootstrap.js`, on line the CLI class was imported as:
```const WebpackCLI = require('./webpack-cli');```
but on line 120, a new instance of the class was initiated by:
```const cli = new webpackCLI();```

**Does this PR introduce a breaking change?**
- No

**Other information**

### CODE REVIEW

**Comment by ematipico** [comment]
It's correct: https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/141e06ef(v=vs.71)?redirectedfrom=MSDN

**Comment by godcrampy** [comment]
Please have a look at the changes again, I am not changing the case. The class is defined as `WebpackCLI` but exported as `webpackCli` thus resulting in an error. I have mentioned about this in the PR summary.

File: lib/bootstrap.js [hunk 1]
```diff
@@ -117,6 +117,6 @@ function isCommandUsed(commands) {
 }
 (async() => {
 	const commandIsUsed = isCommandUsed(commands);
-	const cli = new webpackCli();
+	const cli = new WebpackCLI();
```
Comment (ematipico) [inline lib/bootstrap.js]:
```suggestion
	const cli = new WebpackCli();
```

File: lib/webpack-cli.js [hunk 2]
```diff
@@ -177,4 +177,4 @@ class WebpackCLI extends GroupHelper {
     }
 }
 
-module.exports = webpackCLI;
+module.exports = WebpackCLI;
```
Comment (ematipico) [inline lib/webpack-cli.js]:
```suggestion
module.exports = WebpackCli;
```

**Comment by godcrampy** [comment]
Made requested changes.

**Comment by godcrampy** [comment]
@ematipico I'd be more than happy to help in all the ways I can. BTW i had a few queries:
1. What is the purpose of the `next` branch. How does it compare to `master` branch.
2. 2 days ago @evenstensberg reverted the `next` branch to some commit resulting in a large amount of changes and such minor bugs. Why was this change made?
3. I want to contribute more to webpack! How can I get involved?
4. Also is there any other communication medium you guys use like IRC or something where I can be updated with the dev talks for webpack?

Cheers 😄

**Pull request closed without merging**